### PR TITLE
Fix: Resolve the error caused by the usage of undefined property in the offline donation settings component

### PR DESCRIPTION
**Pull request by alaca**
## Description

This PR resolves the error caused by the usage of undefined property in the offline donation settings component. The issue is resolved by adding the missing property.

## Pre-review Checklist

-   [ ] Acceptance criteria satisfied and marked in related issue
-   [ ] Relevant `@unreleased` tags included in DocBlocks
-   [ ] Includes unit tests
-   [ ] Reviewed by the designer (if follows a design)
-   [x] [Self Review](https://give.gitbook.io/development-manual/devops/github/code-reviews#self-review) of code and UX completed

### CODE REVIEW

File: src/FormBuilder/resources/js/form-builder/src/settings/offline-donation/index.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import DonationInstructions from './donation-instructions';
 
 const OfflineDonationsSettings = () => {
     const {
-        settings: {offlineDonationsCustomize},
+        settings: {enableOfflineDonations, offlineDonationsCustomize},
```
Comment (jonwaldstein) [inline src/FormBuilder/resources/js/form-builder/src/settings/offline-donation/index.js:8]: unless i'm missing something,`offlineDonationsCustomize` does not look like it's being used here - if not let's remove.

Comment (kjohnson) [reply src/FormBuilder/resources/js/form-builder/src/settings/offline-donation/index.js:8]: I resolved this with the merge.

**Comment by kjohnson** [comment]
Merged locally due to (temporary) GitHub status issue with PRs.

**Comment by kjohnson** [comment]
Merged locally due to (temporary) GitHub status issue with PRs.

**Pull request closed without merging**